code/_onclick/hud/alert.dm

```diff
@@ -177,6 +177,41 @@
 	desc = "You're severely malnourished. The hunger pains make moving around a chore."
 	icon_state = "starving"
 
+///Vampire "hunger"
+
+/obj/screen/alert/fat/vampire
```

Comment:
I'm hesitant to do this, and I'll let you know why.

The food indicator is blatant abuse of the alert system; it really shoudln't exist.

This isn't to say we can't have an indicator for food--that's perfectly fine. The thing is, it should be a permanent screen element as opposed to an alert.

Reply:
@Fox-McCloud , I am a bit confused, are you talking about the code work I have done or the hunger indicator system in general?